nelc/ecommerce
# feat: redwood mig/long lastname

## Description

* fix: set more chars for firstname and lastname

This because arabic saml provider set longer this fields.

(cherry picked from commit 675df1ecc49d4190d6fc7dc05c96d055594765f1)

* fix: update migration sequency to avoid problems

(cherry picked from commit 1a71127b0f2021f7d14dcab69fa72e0db6583700)

# Test

- run migrations
![image](https://github.com/user-attachments/assets/f6e3cbf1-ff01-4fe6-8838-333339c30eaf)
- check with a user in the firstname and the lastname you could save more than 30 chars.
![image](https://github.com/user-attachments/assets/eac7b14e-499f-42a1-9e47-ce05eb4d2f7e)
